cms/server/admin/handlers/task.py

```diff
@@ -502,3 +508,43 @@ def delete(self, task_id):
 
         # Maybe they'll want to do this again (for another task)
         self.write("../../tasks")
+
+
+class DownloadSubmissionsHandler(FileHandler):
+    """Download submissions for a dataset in a zip file.
+    """
+    @require_permission(BaseHandler.AUTHENTICATED)
+    def get(self, task_id):
+        task = self.safe_get_item(Task, task_id)
+
+        zip_filename = "%s.zip" % task.name
+        source_template = "%s_%s%s"
+
+        tempdir = tempfile.mkdtemp(dir=config.temp_dir)
+        zip_path = os.path.join(tempdir, zip_filename)
+
+        users = {}
+
+        with zipfile.ZipFile(zip_path, "w") as zip_file:
+            for submission in task.submissions:
+                user = submission.participation.user.username
+                if user not in users:
+                    users[user] = 1
+                else:
+                    users[user] += 1
+                for filename, sub_file in submission.files.items():
```

Comment:
Whoops, didn't see that one. I'm not really sure what to replace it with, but the internet says for unused variables, using `_` is ok, so I'll update the PR with that.